src/utils/time.js

```diff
@@ -1,8 +1,10 @@
 import { format } from "date-fns";
 
 export const ONE_MINUTE_IN_SECONDS = 60;
-const ONE_HOUR_IN_SECONDS = ONE_MINUTE_IN_SECONDS * 60;
-const ONE_DAY_IN_SECONDS = ONE_HOUR_IN_SECONDS * 24;
+export const ONE_HOUR_IN_SECONDS = ONE_MINUTE_IN_SECONDS * 60;
+export const ONE_DAY_IN_SECONDS = ONE_HOUR_IN_SECONDS * 24;
+
+export const FIVE_MINUTES_MS = 5 * 60 * 1000;
```

Comment:
nit: we should be consistent about whether to include `_IN` here:

* `FIVE_MINUTES_IN_MS` vs. `FIVE_MINUTES_MS`
* `ONE_HOUR_IN_SECONDS` vs `ONE_HOUR_SECONDS`

My leaning is toward including `_IN`, but feel free to make the call.

Reply:
I would probably cut `ONE_HOUR_SECONDS` down to `ONE_HOUR_SEC` or similar. I have another WIP PR converting this file to `.ts`, and then probably 🔥 a lot of this, it's super overly verbose, and often implemented in an overly complex way.

eg.
```diff
-export const getCurrentTimeInUTCSeconds = () => {
-  const date = new Date();
-  return (
-    Date.UTC(
-      date.getUTCFullYear(),
-      date.getUTCMonth(),
-      date.getUTCDate(),
-      date.getUTCHours(),
-      date.getUTCMinutes(),
-      date.getUTCSeconds()
-    ) / 1000
-  );
-};
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now
+//   The static Date.now() method returns the number of milliseconds elapsed since January 1, 1970 00:00:00 UTC.
+export const getCurrentTimeInUTCSeconds = () => Date.now() / 1000;
```

Reply:
Perhaps we can also use a linter to disallow `new Date()` and `Date.getTime()`.

Reply:
I was thinking the same myself.

Reply:
backlogged in https://github.com/sparkletown/internal-sparkle-issues/issues/20

Reply:
Going to leave this as is in this PR, we can address it as part of https://github.com/sparkletown/sparkle/pull/897 and/or future. I expect a lot of this stuff may get cut down in future code cleanups/refactors.